2025-05-12 04:48:00,326 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-12 04:48:15,343 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/14161.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/14161.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf28830>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-05-12 04:48:30,359 - ERROR - Request Error http://190.102.43.113/Formigatv/offilinetv2806/1541684.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541684.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf28e30>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:48:45,376 - ERROR - Request Error http://190.102.43.113/Formigatv/offilinetv2806/1541685.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541685.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf29700>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:48:45,396 - ERROR - Request Error http://15.235.15.230:14785/: HTTPConnectionPool(host='15.235.15.230', port=14785): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2a030>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:48:45,415 - ERROR - Request Error http://15.235.15.230:14757/: HTTPConnectionPool(host='15.235.15.230', port=14757): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2a930>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:48:45,435 - ERROR - Request Error http://15.235.15.230:14517/: HTTPConnectionPool(host='15.235.15.230', port=14517): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf29ca0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:48:45,454 - ERROR - Request Error http://15.235.15.230:14519/: HTTPConnectionPool(host='15.235.15.230', port=14519): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf292b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:48:45,473 - ERROR - Request Error http://15.235.15.230:14488/: HTTPConnectionPool(host='15.235.15.230', port=14488): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf28620>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:48:45,492 - ERROR - Request Error http://15.235.15.230:14464/: HTTPConnectionPool(host='15.235.15.230', port=14464): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2ac00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:48:45,512 - ERROR - Request Error http://15.235.15.230:14462/: HTTPConnectionPool(host='15.235.15.230', port=14462): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2b560>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 04:49:00,528 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/14146.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/14146.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf2bef0>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-05-12 04:49:15,544 - ERROR - Request Error http://89.45.12.66:8080/live/toptv_207670/LgNXVm3I/36283.m3u8: HTTPConnectionPool(host='89.45.12.66', port=8080): Max retries exceeded with url: /live/toptv_207670/LgNXVm3I/36283.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf2b290>, 'Connection to 89.45.12.66 timed out. (connect timeout=15)'))
2025-05-12 04:49:30,561 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/1541727.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/1541727.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf28260>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:49:45,577 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320566.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320566.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf28d70>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:50:00,593 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320567.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320567.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf29940>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:50:15,609 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320570.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320570.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf18170>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:50:30,626 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320569.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320569.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf18b00>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:50:45,641 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320563.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320563.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf29400>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:51:00,658 - ERROR - Request Error http://190.102.43.113:80/Formigatv/offilinetv2806/320564.m3u8: HTTPConnectionPool(host='190.102.43.113', port=80): Max retries exceeded with url: /Formigatv/offilinetv2806/320564.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fadfaf28b30>, 'Connection to 190.102.43.113 timed out. (connect timeout=15)'))
2025-05-12 04:51:01,113 - WARNING - URL Error https://cdn-3.nxplay.com.br/GLOBO_SP_TK/index.m3u8: Status Code 403
2025-05-12 04:51:01,375 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-05-12 04:51:01,450 - WARNING - URL Error http://xcui.vitvabc.xyz:8880/laok300/8pEZjdeMPad7/32370: Status Code 401
2025-05-12 04:51:01,518 - WARNING - URL Error http://xcui.vitvabc.xyz:8880/laok300/8pEZjdeMPad7/32266: Status Code 401
2025-05-12 04:51:19,650 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112659
2025-05-12 04:51:37,361 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112660
2025-05-12 04:51:53,371 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112664
2025-05-12 04:52:11,390 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112665
2025-05-12 04:52:12,075 - WARNING - URL Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112666: Status Code 403
2025-05-12 04:52:41,613 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112667: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:52:58,607 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112668: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:53:14,387 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112669: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:53:31,584 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112670: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:53:47,363 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112671: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:54:04,451 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112672: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:54:21,392 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112673: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:54:38,307 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112674: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:54:54,088 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112675: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:55:11,259 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112676: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:55:27,039 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112677: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:55:42,833 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112678: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:55:59,929 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112679: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:56:16,920 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112680: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:56:33,934 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112681: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 04:56:49,719 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112682: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:57:05,524 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112683: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:57:21,313 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112684: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out. (read timeout=15)
2025-05-12 04:57:37,640 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112685
2025-05-12 04:57:54,649 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112686
2025-05-12 04:58:10,659 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112687
2025-05-12 04:58:27,674 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112688
2025-05-12 04:58:44,687 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112689
2025-05-12 04:58:50,696 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112690
2025-05-12 04:59:01,456 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112691
2025-05-12 04:59:19,479 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112692: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 05:00:11,682 - ERROR - Request Error http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112693: HTTPConnectionPool(host='146.0.35.86', port=8065): Read timed out.
2025-05-12 05:00:14,754 - INFO - URL OK: http://callvip.me:80/DodaLiin/jXcQY5EUFzcqjn72PTRD/112694
2025-05-12 05:00:14,773 - ERROR - Request Error http://15.235.15.230:14783/: HTTPConnectionPool(host='15.235.15.230', port=14783): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2b830>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,793 - ERROR - Request Error http://15.235.15.230:14478/: HTTPConnectionPool(host='15.235.15.230', port=14478): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2baa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,812 - ERROR - Request Error http://15.235.15.230:14492/: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf28d10>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,831 - ERROR - Request Error http://15.235.15.230:14485/: HTTPConnectionPool(host='15.235.15.230', port=14485): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf29790>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,850 - ERROR - Request Error http://15.235.15.230:14495/: HTTPConnectionPool(host='15.235.15.230', port=14495): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf29fa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,870 - ERROR - Request Error http://15.235.15.230:14502/: HTTPConnectionPool(host='15.235.15.230', port=14502): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf19b80>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,889 - ERROR - Request Error http://15.235.15.230:14506/: HTTPConnectionPool(host='15.235.15.230', port=14506): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf28560>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,909 - ERROR - Request Error http://15.235.15.230:14504/: HTTPConnectionPool(host='15.235.15.230', port=14504): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf29a00>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,928 - ERROR - Request Error http://15.235.15.230:14508/: HTTPConnectionPool(host='15.235.15.230', port=14508): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2aa20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,947 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf29190>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,967 - ERROR - Request Error http://15.235.15.230:14512/: HTTPConnectionPool(host='15.235.15.230', port=14512): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfb519940>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:14,986 - ERROR - Request Error http://15.235.15.230:14510/: HTTPConnectionPool(host='15.235.15.230', port=14510): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf1b860>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,005 - ERROR - Request Error http://15.235.15.230:14516/: HTTPConnectionPool(host='15.235.15.230', port=14516): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf28fb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,025 - ERROR - Request Error http://15.235.15.230:14492/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14492): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf29e80>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,044 - ERROR - Request Error http://15.235.15.230:14487/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf28770>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,064 - ERROR - Request Error http://15.235.15.230:14499/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14499): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2a6f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,083 - ERROR - Request Error http://15.235.15.230:14513/AFDTV: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: /AFDTV (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf188f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,102 - ERROR - Request Error http://15.235.15.230:14513/: HTTPConnectionPool(host='15.235.15.230', port=14513): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf1a660>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:15,346 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-12 05:00:15,610 - WARNING - URL Error http://s1.1-cup.net:2095/solucao111hgfgh77hu78/xplus1111XXhjgyu87/51: Status Code 404
2025-05-12 05:00:15,630 - ERROR - Request Error http://15.235.15.230:14487/: HTTPConnectionPool(host='15.235.15.230', port=14487): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fadfaf2bdd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-05-12 05:00:16,120 - INFO - URL OK: https://cdn-3.nxplay.com.br/GLOBO_SP_TK/tracks-v1a1/mono.m3u8
2025-05-12 05:00:16,387 - INFO - URL OK: http://hls1.sua.tv/live/globotvbahiafhdbr2/s.m3u8
2025-05-12 05:00:16,624 - INFO - URL OK: http://hls1.sua.tv:80/live/globosphd/s.m3u8
2025-05-12 05:00:16,684 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/90059.m3u8: Status Code 404
2025-05-12 05:00:16,741 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/91431.m3u8: Status Code 404
2025-05-12 05:00:16,880 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/90034.m3u8: Status Code 404
2025-05-12 05:00:16,943 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91421.m3u8: Status Code 404
2025-05-12 05:00:16,991 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91435.m3u8: Status Code 404
2025-05-12 05:00:17,044 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91463.m3u8: Status Code 404
2025-05-12 05:00:17,100 - WARNING - URL Error http://metv.fun:80/4x6cXF/JDqRqj/90064.m3u8: Status Code 404
2025-05-12 05:00:17,159 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/90019.m3u8: Status Code 404
2025-05-12 05:00:17,207 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95441.m3u8: Status Code 404
2025-05-12 05:00:17,266 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91405.m3u8: Status Code 404
2025-05-12 05:00:17,308 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91413.m3u8: Status Code 404
2025-05-12 05:00:17,361 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/120082.m3u8: Status Code 404
2025-05-12 05:00:17,405 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91443.m3u8: Status Code 404
2025-05-12 05:00:17,463 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91445.m3u8: Status Code 404
2025-05-12 05:00:17,516 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95483.m3u8: Status Code 404
2025-05-12 05:00:17,564 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/97396.m3u8: Status Code 404
2025-05-12 05:00:17,614 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91456.m3u8: Status Code 404
2025-05-12 05:00:17,671 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91460.m3u8: Status Code 404
2025-05-12 05:00:17,724 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/95448.m3u8: Status Code 404
2025-05-12 05:00:17,776 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91461.m3u8: Status Code 404
2025-05-12 05:00:17,829 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91467.m3u8: Status Code 404
2025-05-12 05:00:17,879 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91470.m3u8: Status Code 404
2025-05-12 05:00:17,934 - WARNING - URL Error http://lidns.cc:80/4x6cXF/JDqRqj/91471.m3u8: Status Code 404
